### DISCUSSION TITLE
libbitsandbytes_cpu.so: undefined symbol: cget_col_row_stats

### DISCUSSION THREAD

**Discussion by qixing-ai** [LAION-AI/Open-Assistant#3468] [Q&A]
I get this error when I test the model with the following command:
python tools/model_cli.py --model_path model

<img width="1144" alt="image" src="https://user-images.githubusercontent.com/16987074/222363818-d7a98ee0-1b73-42ab-ba00-6760341fff68.png">

**Comment by silvacarl2** [comment]
same here.  Did you find a solution?

**Comment by silvacarl2** [comment]
use this:

# ImportError : undefined symbol: cudaLaunchKernelExC  version 

cp /home/silvacarl/.local/lib/python3.8/site-packages/bitsandbytes/libbitsandbytes_cuda117.so /home/silvacarl/.local/lib/python3.8/site-packages/bitsandbytes/libbitsandbytes_cpu.so
